src/frontend/components/StudentLiveWrapper/index.tsx

```diff
@@ -53,8 +53,13 @@ export const LiveVideoWrapper: React.FC<LiveVideoWrapperProps> = ({
   useEffect(() => {
     // if the xmpp object is not null, panel state is filled
     if (video.xmpp !== null) {
+      const items = [];
+      if (video.has_chat) {
+        items.push(LivePanelItem.CHAT);
+      }
+      items.push(LivePanelItem.VIEWERS_LIST);
```

Comment:
```suggestion
      const items = [LivePanelItem.VIEWERS_LIST];
      if (video.has_chat) {
        items.push(LivePanelItem.CHAT);
      }
```
Same suggestion as above

Reply:
the answer Vassili gave also applies here